src/sentry/tagstore/multi/backend.py

```diff
@@ -49,11 +60,32 @@ def worker():
         t.start()
 
     def run(self, f, *args, **kwargs):
+        metrics.incr(
+            'tagstore.multi.runner',
+            instance='run',
+            skip_internal=True,
+        )
         if random.random() <= options.get('tagstore.multi-sampling'):
             try:
                 self.q.put((f, args, kwargs), block=False)
+                metrics.incr(
+                    'tagstore.multi.runner',
```

Comment:
I'd append a `.schedule` to these three to separate the scheduling from the executing.